client/my-sites/site-indicator/site-indicator.jsx

```diff
@@ -295,4 +357,32 @@ export default React.createClass( {
 			</div>
 		);
 	}
-} );
+
+	render() {
+		const { site, siteIsJetpack } = this.props;
+
+		return (
+			<div className="site-indicator__wrapper">
+				{ siteIsJetpack && <QuerySiteConnectionStatus siteId={ site.ID } /> }
```

Comment:
Will that make any difference really? And it doesn't make much sense to pass it to the component IMO - it just doesn't seem natural to me.